make bootstrap fails when Installing TrezorCrypto

when I use `make bootstrap` command, on OSX 10.14 and Xcode 10.1
```bash
bundle exec pod install --repo-update
Updating local specs repositories
  $ /usr/local/bin/git -C /Users/ludorum/.cocoapods/repos/master fetch origin --progress
  remote: Enumerating objects: 28, done.        
  remote: Counting objects: 100% (28/28), done.        
  remote: Compressing objects: 100% (13/13), done.        
  remote: Total 18 (delta 12), reused 7 (delta 4), pack-reused 0        
  From https://github.com/CocoaPods/Specs
     380f7a3b1f7..8834e53db0b  master     -> origin/master
  $ /usr/local/bin/git -C /Users/ludorum/.cocoapods/repos/master rev-parse --abbrev-ref HEAD
  master
  $ /usr/local/bin/git -C /Users/ludorum/.cocoapods/repos/master reset --hard origin/master
  HEAD is now at 8834e53db0b [Add] SwiftyXMLParser 4.3.0

CocoaPods 1.6.0.beta.2 is available.
To update use: `sudo gem install cocoapods --pre`
[!] This is a test version we'd love you to try.

For more information, see https://blog.cocoapods.org and the CHANGELOG for this version at https://github.com/CocoaPods/CocoaPods/releases/tag/1.6.0.beta.2

Analyzing dependencies
Pre-downloading: `JSONRPCKit` from `https://github.com/bricklife/JSONRPCKit.git`
Pre-downloading: `QRCodeReaderViewController` from `https://github.com/yannickl/QRCodeReaderViewController.git`, branch `master`
Pre-downloading: `TrustCore` from `https://github.com/TrustWallet/trust-core`, branch `master`
Pre-downloading: `TrustKeystore` from `https://github.com/TrustWallet/trust-keystore`, branch `master`
Pre-downloading: `TrustWalletSDK` from `https://github.com/TrustWallet/TrustSDK-iOS`, branch `master`
Pre-downloading: `TrustWeb3Provider` from `https://github.com/TrustWallet/trust-web3-provider`, commit `f4e0ebb1b8fa4812637babe85ef975d116543dfd`
Downloading dependencies
Installing APIKit (3.2.1)
Installing Alamofire (4.7.3)
Installing BigInt (3.1.0)
Installing Branch (0.25.8)
Installing CryptoSwift (0.10.0)
Installing Eureka (4.3.0)
Installing JSONRPCKit (3.0.0)
Installing KeychainSwift (13.0.0)
Installing Kingfisher (4.10.1)
Installing MBProgressHUD (1.1.0)
Installing Moya (10.0.2)
Installing PromiseKit (6.5.2)
Installing QRCodeReaderViewController (4.0.2)
Installing R.swift (4.0.0)
Installing R.swift.Library (4.0.0)
Installing Realm (3.11.2)
Installing RealmSwift (3.11.2)
Installing Result (3.2.4)
Installing SAMKeychain (1.5.3)
Installing SeedStackViewController (0.4.0)
Installing SipHash (1.2.2)
Installing StatefulViewController (3.0)
Installing SwiftLint (0.28.0)
Installing SwiftProtobuf (1.2.0)
Installing TrezorCrypto (0.0.9)
[!] /bin/bash -c 
set -e
git submodule update --init
sed -i '' -e 's:ed25519-donna/::g' ./**/*.c
sed -i '' -e 's:ed25519-donna/::g' ./**/*.h
sed -i '' -e 's:USE_ETHEREUM 0:USE_ETHEREUM 1:g' trezor-crypto/options.h

sed: can't read : No such file or directory

make: *** [install] Error 1
```

--- a/Trust.xcodeproj/project.pbxproj
+++ b/Trust.xcodeproj/project.pbxproj
@@ -2700,7 +2700,7 @@
 			files = (
 			);
 			inputPaths = (
-				"${SRCROOT}/Pods/Target Support Files/Pods-Trust/Pods-Trust-resources.sh",
+				"${PODS_ROOT}/Target Support Files/Pods-Trust/Pods-Trust-resources.sh",
 				"${PODS_CONFIGURATION_BUILD_DIR}/TrustWeb3Provider/TrustWeb3Provider.bundle",
 			);
 			name = "[CP] Copy Pods Resources";
@@ -2709,7 +2709,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-Trust/Pods-Trust-resources.sh\"\n";
+			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-Trust/Pods-Trust-resources.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
 		29282B511F7629DB0067F88D /* Run SwiftLint */ = {
@@ -2782,7 +2782,7 @@
 			files = (
 			);
 			inputPaths = (
-				"${SRCROOT}/Pods/Target Support Files/Pods-Trust/Pods-Trust-frameworks.sh",
+				"${PODS_ROOT}/Target Support Files/Pods-Trust/Pods-Trust-frameworks.sh",
 				"${BUILT_PRODUCTS_DIR}/APIKit/APIKit.framework",
 				"${BUILT_PRODUCTS_DIR}/Alamofire/Alamofire.framework",
 				"${BUILT_PRODUCTS_DIR}/BigInt/BigInt.framework",
@@ -2841,7 +2841,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-Trust/Pods-Trust-frameworks.sh\"\n";
+			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-Trust/Pods-Trust-frameworks.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
 		F48CC1C9F6D4FAE6EF587C03 /* [CP] Check Pods Manifest.lock */ = {

